code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -80,6 +80,38 @@
 
 	. += config.human_delay
 
+	Process_SL_Locator() //Adjust the angle of the overlay
+
+
+/mob/living/carbon/human/proc/Process_SL_Locator()
+	if(!sl_headset_active)
+		return
+	if(!assigned_squad)
+		return
+	if(world.time < sl_locator_next_update)
```

Comment:
I'd place this check first. It would probably cover the other two cases since `sl_locator_next_update` wouldn't be updated anyway.